Sources/WalletConnectNotify/Client/Wallet/NotifyClient.swift

```diff
@@ -86,8 +86,8 @@ public class NotifyClient {
         self.notifySubscriptionsChangedRequestSubscriber = notifySubscriptionsChangedRequestSubscriber
     }
 
-    public func register(account: Account, onSign: @escaping SigningCallback) async throws {
-        _ = try await identityClient.register(account: account, onSign: onSign)
+    public func register(account: Account, domain: String, isLimited: Bool, onSign: @escaping SigningCallback) async throws {
```

Comment:
should we make isLimited=false by default?

Reply:
let's do